packages/react/src/components/Toolbar/ToolbarMenuRadioGroup.tsx

```diff
@@ -0,0 +1,147 @@
+import * as customPropTypes from '@stardust-ui/react-proptypes'
+import * as _ from 'lodash'
+import * as PropTypes from 'prop-types'
+import * as React from 'react'
+
+import {
+  ChildrenComponentProps,
+  ContentComponentProps,
+  createShorthandFactory,
+  UIComponentProps,
+  commonPropTypes,
+  ShorthandFactory,
+  applyAccessibilityKeyHandlers,
+  UIComponent,
+} from '../../lib'
+import {
+  ComponentEventHandler,
+  ShorthandCollection,
+  ShorthandValue,
+  WithAsProp,
+  withSafeTypeForAs,
+} from '../../types'
+import { Accessibility } from '../../lib/accessibility/types'
+import ToolbarMenuItem, { ToolbarMenuItemProps } from './ToolbarMenuItem'
+import { mergeComponentVariables } from '../../lib/mergeThemes'
+import Box, { BoxProps } from '../Box/Box'
+import {
+  toolbarMenuRadioGroupBehavior,
+  toolbarMenuItemRadioBehavior,
+} from '../../lib/accessibility'
+
+export interface ToolbarMenuRadioGroupProps
+  extends UIComponentProps,
+    ChildrenComponentProps,
+    ContentComponentProps {
+  /**
+   * Accessibility behavior if overridden by the user.
+   */
+  accessibility?: Accessibility
+
+  /** Index of the currently active item. */
+  activeIndex?: number
```

Comment:
to meke it predictable, I think we should add activeIndex to toolbarRadioGroup as well

Reply:
Added 👍